compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/reduce-reactive-deps/jump-unpoisoned/throw-before-scope-starts.expect.md

```diff
@@ -75,7 +75,7 @@ export const FIXTURE_ENTRYPOINT = {
       
 ### Eval output
 (kind: ok) [[ (exception in render) Error: throw with error! ]]
-[[ (exception in render) Error: throw with error! ]]
+[2]
```

Comment:
Note that we now correctly do the non-throwing re-render instead of returning a cached error